conman/routes/handlers.py

```diff
@@ -1,4 +1,7 @@
-from django.core.urlresolvers import resolve
+from conman.compat import ignore_django20_deprecation
+
+with ignore_django20_deprecation():
+    from django.core.urlresolvers import resolve
```

Comment:
Rather than suppress the warning, could we instead do:

```python
try:
    from django.urls import resolve
except ImportError:
    from django.core.urlresolvers import resolve
```

...or alternatively add the import to the `compat` module, and import from there?